doc/JSON-RPC-interface.md

```diff
@@ -62,6 +62,8 @@ bitcoin-cli -named createwallet wallet_name=mywallet load_on_startup=true
 bitcoin-cli -named createwallet mywallet load_on_startup=true
 ```
```

Comment:
In "doc: Mention bitcoin wrapper executable in documentation"  b0989e3095f1964fdb523cdd4b44f0b198417014

I think it would be better to provide a single usage instruction for users. Otherwise, having separate instructions for both `bin/bitcoin` and `bin/bitcoind` in all places could be confusing.

Instead, I suggest we standardize on `bin/bitcoin` in all places, while still supporting the previous one for backward compatibility. 

The release notes have clearly indicate this change to users.

Reply:
> It seems like it could be more confusing to not mention one of the alternatives than just describe both.


You are right I think it's okay to include both alternative comment in the `doc/README.md`.

However In my opinion, that's sufficient, and we don't need to repeat in every instance that, for example, `bitcoin node` or `bitcoin GUI` can be substituted with `bitcoind`, etc.